var/spack/repos/builtin/packages/roctracer-dev/package.py

```diff
@@ -42,7 +46,7 @@ def patch(self):
                     string=True)
         kwargs = {'ignore_absent': False, 'backup': False, 'string': False}
         with working_dir('script'):
-            match = '^#!/usr/bin/python'
+            match = '^#!/usr/bin/python[2]'
```

Comment:
is this from upstream? shouldn't `[2] == 2` for regular expressions? Seems very fishy.

Reply:
yeah...I have just checked out this from upstream. I will take a look at why this is done this way.